hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoop/HadoopPlugin.groovy

```diff
@@ -94,7 +94,9 @@ class HadoopPlugin implements Plugin<Project> {
 
       // Add clusterProvided jars on the classpath, this will not add transitive dependencies of each jar
       project.sourceSets.main.compileClasspath += clusterProvided
-      project.javadoc.classpath += clusterProvided
+      if (project.javadoc.hasProperty("classpath")) {
```

Comment:
@ethankhall javadoc task according to gradle is used for generating documentation for classes. I don't know if that's being done here by setting a classpath's config here. Let me know if having this makes sense. Removing this doesn't seem to have making any difference.